src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -172,7 +172,13 @@ public function section($key, $template=null)
 
 		if ($template === null)
 		{
-			$template = 'block_section';
```

Comment:
Removing this line is wrong. If the foreach loop does not find a template, we'd end up with `null`.